pkg/cloud/services/ec2/instances.go

```diff
@@ -1008,6 +1022,56 @@ func (s *Service) ModifyInstanceMetadataOptions(instanceID string, options *infr
 	return nil
 }
 
+// GetDHCPOptionSetDomainName returns the domain DNS name for the VPC from the DHCP Options.
+func (s *Service) GetDHCPOptionSetDomainName(ec2client ec2iface.EC2API, vpcID *string) *string {
+	fmt.Println("vpcID", vpcID)
```

Comment:
```suggestion
```
Leftover print, if we want to keep it we could maybe log it at a debug level below the nil check